src/web/EmployerFavouritesWeb/Domain/WriteModel/ApprenticeshipFavourite.cs

```diff
@@ -1,31 +1,32 @@
 using System.Collections.Generic;
+using System.Linq;
 
 namespace DfE.EmployerFavourites.Domain.WriteModel
 {
     public class ApprenticeshipFavourite
     {
         public ApprenticeshipFavourite()
         {
-            Ukprns = new List<int>();
+            Providers = new List<Provider>();
         }
 
         public ApprenticeshipFavourite(string apprenticeshipId) : this()
         {
             ApprenticeshipId = apprenticeshipId;
         }
 
-        public ApprenticeshipFavourite(string apprenticeshipId, IList<int> ukprns) : this(apprenticeshipId)
+        public ApprenticeshipFavourite(string apprenticeshipId, IDictionary<int,IList<int>> providers) : this(apprenticeshipId)
         {
-            if (ukprns != null)
-                Ukprns = ukprns;
+            if (providers != null)
+                Providers = providers.Select(p => new Provider (p.Key,p.Value )).ToList();
         }
 
-        public ApprenticeshipFavourite(string apprenticeshipId, int ukprn) : this(apprenticeshipId)
-        {
-            Ukprns.Add(ukprn);
-        }
+        //public ApprenticeshipFavourite(string apprenticeshipId, Provider provider) : this(apprenticeshipId)
```

Comment:
Can this be removed?

Reply:
Yes, it was restructured to use a dictionary